docs/src/main/tut/user-guide.md

```diff
@@ -886,6 +886,31 @@ val bufEnumerator =
   }
 ```
 
+The `finch-refined` module provides support for [refined][refined] types in path segments, query parameters and
+other request entities. This approach enables validation of API on type level:
+
+```tut
+import java.net.URL
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.numeric._
+import eu.timepit.refined.string._
+import io.finch._
+import io.finch.syntax._
+
+val e: Endpoint[Int] = get("foo" :: param[Int Refined Positive]("int")) { (i: Int Refined Positive) =>
+  Ok(i.value)
+}
+
+val u: Endpoint[URL] = get("foo" :: param[String Refined Url]("url")) { (s: String Refined Url) =>
+  Ok(new URL(s.value))
+}
+
+//Throw(io.finch.Error.NotParsed: param 'int' cannot be converted to Refined: Predicate failed: (-1 > 0)..)
```

Comment:
Probably don't need this comment here. `tut` will render the resulting `Throw` in the docs.

Reply:
Sure, thanks